treeherder/model/sql/jobs.json

```diff
@@ -418,28 +417,17 @@
             "host_type":"read_host"
 
         },
-        "get_max_job_submit_timestamp":{
+        "get_max_job_start_timestamp":{
```

Comment:
The result here is used to generate the time window ([see here](https://github.com/mozilla/treeherder/blob/0f3d1b2bcf8700070e7e99892bf1990528811bd0/treeherder/model/derived/jobs.py#L569-L578)) that is passed into `get_running_eta_stats` for the `>=` WHERE clause. Since we no longer care about pending ETAs, the `submit_timestamp` is irrelevant (since: "running time = end - start" ; ie doesn't depend on submit time).